pkg/gits/git_cli.go

```diff
@@ -520,8 +520,13 @@ func (g *GitCLI) AddCommit(dir string, msg string) error {
 
 // AddCommitFiles perform an add and commit selected files from the repository at the given directory with the given messages
 func (g *GitCLI) AddCommitFiles(dir string, msg string, files []string) error {
```

Comment:
Why not use `Add` followed by `CommitDir`? We are adding all these very specific functions making it really hard to reuse any of this and also making it hard to understand/read the code making use of it, since we are not using/mirroring standard git CLI workflows.

Reply:
This is a change to an existing function. It's aim is to allow adding of specific files to a commit, doing an add may include unintended files to the commit.